src/deploy-cromwell-on-azure/Configuration.cs

```diff
@@ -44,6 +50,10 @@ public abstract class UserAccessibleConfiguration
         public string LogAnalyticsArmId { get; set; }
         public string ApplicationInsightsAccountName { get; set; }
         public string VmName { get; set; }
+        public bool UseAks { get; set; }
+        public string AksClusterName { get; set; }
```

Comment:
I believe AKS configuration is multually exclusive to the VM configuration; for example, if people use AKS, they don't need to supply VM configuration like VM name etc. Is there a better way to convey it in the config or atleast call out through documentation/comments